tests/unittest/llmapi/apps/_test_trtllm_serve_profile.py

```
@@ -0,0 +1,207 @@
+                    pass
+
+        ticker_task = asyncio.create_task(_ticker())
+        # Warm up the ticker before invoking the handler so we have a
+        # cadence baseline.
+        await asyncio.sleep(0.05)
+        await handler_call()
+        await asyncio.sleep(0.05)
+        stop_event.set()
+        await ticker_task
+
+        max_gap = max(b - a for a, b in zip(ticks, ticks[1:])) if len(ticks) > 1 else 0.0
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)